MotorRequest motor1
MotorRequest motor2
MotorRequest motor3
MotorRequest motor4

float32 duration
